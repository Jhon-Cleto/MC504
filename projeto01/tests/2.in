10 1
1 10
5342