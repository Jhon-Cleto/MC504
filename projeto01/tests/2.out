Multiplicação de Matrizes: A x B = C

Matriz A (10 x 1):
 004
 003
 009
 003
 001
 002
 001
 008
 005
 000

Matriz B (1 x 10):
 001 008 004 006 005 002 007 001 006 005

Matriz C (10 x 10):
 004 032 016 024 020 008 028 004 024 020
 003 024 012 018 015 006 021 003 018 015
 009 072 036 054 045 018 063 009 054 045
 003 024 012 018 015 006 021 003 018 015
 001 008 004 006 005 002 007 001 006 005
 002 016 008 012 010 004 014 002 012 010
 001 008 004 006 005 002 007 001 006 005
 008 064 032 048 040 016 056 008 048 040
 005 040 020 030 025 010 035 005 030 025
 000 000 000 000 000 000 000 000 000 000
